ts/src/pro/bybit.ts

```diff
@@ -175,7 +175,8 @@ export default class bybit extends bybitRest {
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const messageHash = 'ticker:' + market['symbol'];
+        symbol = market['symbol'];
+        const messageHash = 'ticker:' + symbol;
         const url = this.getUrlByMarketType (symbol, false, params);
```

Comment:
@carlosmiei Oh, I am sorry. `symbol` value of this line will be different. But by looking in `getUrlByMarketType`, it also will be resolved to the actual symbol.

Reply:
in that case, is it ok to close the PR? 🙂

Reply:
I still think this is more consistent because other exchanges usually written in this pattern and don't use the given `symbol` again after calling `this.market (symbol)`.
But if you think this is not worth to do, please feel free to close it.

Reply:
@youknowone So, the change here is just to use `symbol` (after unifying it) instead of `market['symbol']` ?

Reply:
Including it, the value passing to `getUrlByMarketType` is also changed.

The final behavior is not changed. (my first title and description was wrong)